style.css

```diff
@@ -1,55 +1,66 @@
-.box div {
-    width: 100px;
-    height: 100px;
-    margin: 10px;
-    background: #ff00ff;
+body {
+    color: #333;
+    font-family: Arial, Helvetica, sans-serif;
+}
+p {
+    font-weight: bold;
+}
+h1 {
+    font-style: italic;
+}
+.th {
+    font-variant-caps: all-petite-caps;
+}
+th {
+    background-color: #0066ff;
+}
+td {
+    background-color: #00aeff;
+}
+a {
+    color: #00f;
+    text-decoration-line: underline;
+}
+a:visited {
+    color: #005;
+    text-decoration-line: underline;
+}
+a:hover {
+    text-decoration-line: none;
+}
+ul {
+    list-style: square;
     float: right;
+    margin-top: -9px;
+}
+.border {
+    text-decoration: 1px underline solid red;
```

Comment:
what it this?
does it work?
there in such values for `text-decoration` properties and the name of a class is `BORDER` so only border property should be assigned